recipe/meta.yaml

```diff
@@ -126,7 +127,10 @@ requirements:
     - libwebp
   run_constrained:
     - __osx >={{ MACOSX_DEPLOYMENT_TARGET|default("10.14") }}  # [osx and x86_64]
-    - qt {{ version }}
+    # Qt 5.15.3 was only ever released for commercial users.
+    # https://www.qt.io/blog/commercial-lts-qt-5.15.3-released
```

Comment:
Apparently qt-base 5.15.3 was released earlier this year
https://lists.qt-project.org/pipermail/development/2022-March/042262.html